lnbits/core/db.py

```diff
@@ -0,0 +1,5 @@
+from lnbits.core.models import CoreAppExtra
+from lnbits.db import Database
+
+db = Database("database")
+core_app_extra: CoreAppExtra = CoreAppExtra()
```

Comment:
`core_app_extra` just doesnt really fit here. Not really sure where it might be better tough. Maybe the two methods behind it could somehow become helpers instead?

Reply:
- not sure if this file should exist

Reply:
without that file, you would have a circular import again